schemas/oic.r.coapcloudconf-update-schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
+  "id": "https://www.openconnectivity.org/ocf-apis/core/schemas/oic.r.coapcloudconf-update-schema.json#",
+  "definitions": {
+    "oic.r.coapcloudconf": {
+      "type": "object",
+      "properties": {
+        "apn": {
+          "type": "string",
+          "description": "Authentication provider's name issuing the auth code",
+          "pattern": "^.*$"
+        },
+        "cis": {
+          "type": "string",
+          "description": "Indicates URL for CoAP native cloud interface server to be registered",
+          "format": "uri"
+        },
+        "at": {
+          "type": "string",
+          "description": "Indicates access token which is given in return of auth code issued by an appropriate account server",
+          "pattern": "^.*$"
+        },
+        "att": {
+          "enum": ["Unknown", "Bearer", "MAC"],
+          "description": "Indicates access token type"
+        },
+        "sid": {
+          "type": "string",
+          "description": "The Server identity of the OCF Cloud used for verifying the OCF Cloud certificate.",
+          "pattern": "^.*$"
+        },
+        "trustca":   {
+          "allOf": [
+            {
+              "type": "object",
+              "description": "Trusted Certificate of the OCF cloud to be connected."
+            },
+            {
+              "$ref": "../../security-models/schemas/oic.sec.pubdatatype.json#/definitions/oic.sec.pubdatatype"
+            }
+          ]
+        }
+      },
+      "required":["cis"]
```

Comment:
Required should also include "at" and maybe "sid"

In fact, I think the same Properties are required for both UPDATE and RETRIEVE.  If so, we don't need the UPDATE schema... unless we make "at" write only.